src/colorizer/TimeControls.ts

```diff
@@ -1,54 +1,35 @@
 import ColorizeCanvas from "./ColorizeCanvas";
 
+// TODO: Remove class?
+
 // time / playback controls
 const DEFAULT_TIMER_ID = -1;
 export default class TimeControls {
-  private playBtn: HTMLButtonElement;
-  private pauseBtn: HTMLButtonElement;
-  private forwardBtn: HTMLButtonElement;
-  private backBtn: HTMLButtonElement;
-  private timeSlider: HTMLInputElement;
-  private timeInput: HTMLInputElement;
-
+  // TODO: Change to be React state?
```

Comment:
Same comment I had for RecordingControls, it was almost structured to be its own component already!